frontend/package.json

```diff
@@ -28,7 +28,7 @@
         "@heroicons/react": "^1.0.6",
         "@monaco-editor/react": "^4.5",
         "@primer/octicons-react": "^17",
-        "@redpanda-data/ui": "^3.17.2",
+        "@redpanda-data/ui": "^3.21.0",
```

Comment:
For the latest UI release (3.24.0 at the time of writing this), there have been changes to the Text component as well as the color palette, please make sure that the look & feel is still good.